code/style.css

```diff
@@ -1,7 +1,227 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #ededed;
+}
+
+
+h1 {
+  font-family: 'Playfair Display', serif;
+  font-size: 24px;
+  text-align: center;
+}
+
+h2 {
+  font-family: 'Playfair Display', serif;
+  font-size: 20px;
+}
+
+p {
+  font-family: 'Roboto', sans-serif;  
+}
+
+/* HEADER */
+
+.header-background-image {
+  height: 400px;
+  background-image: url("img/SannaDolckWall_Wedding2_640px.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.header-background-image h1 {
+  color: #13443b;
+  margin-top: -100px;
+  text-align: center;
+  font-size: 32px;
+}
+
+.header-background-image h2 {
+  color: #7e947f;
+  margin-top: -10px;
```

Comment:
Negative margins can usually be achieved in another way.  Try removing your negative margins and setting 
`line-height: 18px` on your `h1` above.